### PR TITLE
Publishing a new module for eepsa ontology: bim4eepsa

### PR DESCRIPTION
**Pull request by iesnaola**
Publishing a new module for eepsa ontology, with information regarding buildiings

### CODE REVIEW

**Comment by davidlehn** [comment]
The target URL doesn't exist.  Is that URL correct?

**Comment by iesnaola** [comment]
Sorry, my bad. I fixed the URL

File: measurements4eepsa/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
++##The Measurements for EEPSA (Energy Efficiency Prediction Semantic Assistant) Ontology
+ +
+ +Contact
+ +
+ +Iker Esnaola-Gonzalez (iker.esnaola AT tekniker.es)
```
Comment (davidlehn) [inline measurements4eepsa/README.md]: You've got extra leading "+" chars here.

File: measurements4eepsa/.htaccess [hunk 2]
```diff
@@ -0,0 +1,5 @@
++Header set Access-Control-Allow-Origin *
+ +Header set Access-Control-Allow-Headers DNT,X-Mx-ReqToken,Keep-Alive,User-Agent,X-Requested-With,If-Modified$
+ +Options +FollowSymLinks
+ +RewriteEngine on
+ +RewriteRule ^$ http://193.144.237.227:8890/DAV/home/dba/Ontologies/measurements4EEPSA.owl [R=302,L]
```
Comment (davidlehn) [inline measurements4eepsa/.htaccess]: Extra leading "+" chars.

**Comment by iesnaola** [comment]
Sorry. I hope this time it works.

**Pull request merged**